deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/CircuitBreakerConfigurationParser.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+
+import io.quarkiverse.openapi.generator.deployment.circuitbreaker.CircuitBreakerConfiguration.CircuitBreakerClassConfiguration;
+
+public final class CircuitBreakerConfigurationParser {
+
+    private final UnaryOperator<String> nameToValuePropertyMapper;
+
+    public CircuitBreakerConfigurationParser(UnaryOperator<String> nameToValuePropertyMapper) {
```

Comment:
I think it will be better to have a constructor that accepts both the UnaryOperator and the Collection of propery names.